src/Data/List/Literals.agda

```diff
@@ -14,8 +14,9 @@ open import Agda.Builtin.Char using (Char)
 open import Agda.Builtin.List using (List)
 open import Data.String.Base using (toList)
 
-isString : IsString (List Char)
-isString = record
-  { Constraint = λ _ → ⊤
-  ; fromString = λ s → toList s
-  }
+instance
```

Comment:
So I think this is backwards (unfortunately). We want instances declared in `Data.List.Instances` so that people who don't want them can avoid them entirely.

We do want `isString` defined here - just not as an instance.